zellij-client/src/lib.rs

```diff
@@ -168,6 +168,7 @@ pub fn start_client(
             colors: palette,
             rounded_corners: config.ui.pane_frames.rounded_corners,
             hide_session_name: config.ui.pane_frames.hide_session_name,
+            invert_selection_colors: config_options.invert_selection_colors.unwrap_or_default(),
```

Comment:
This was initially confusing for me... why do we pass in `config` and `config_options` separately? I originally accessed this via `config.options` but apparently the CLI options flags aren't sync'd to this. I spent quite a bit of time scratching my head. Maybe some newtype safety can help here if this separation is intentional.